### ISSUE TITLE
my gh-pages does not automatically updated.

### ISSUE THREAD

**Issue by azzamsa** [barryclark/jekyll-now#940]
I have a site [here](http://life.azzamsa.com/), and i put the source file [here](https://github.com/azzamsa/azzamsa-life)
when I push my commit into master-branch the site (life.azzamsa.com) does not updated, I have to manually commit to gh-pages branch. But the manual said that we don't have to commit to gh-pages, github does it for us, we just need to push to master branch.

anything wrong with my setting ?

**Comment by sreetamdas** [comment]
Hello @azzamsa 
Is your repo private? Please check your link.

**Comment by azzamsa** [comment]
@sreetamdas  I am sorry, I have a typo in the link. 

Really thanks for the response.

**Comment by sreetamdas** [comment]
could you please check in your repo settings, under the "GitHub Pages" subheading, what's the source?

If it is set as `gh-pages`, change it to `master` and that should resolve your problem right away

**Comment by azzamsa** [comment]
@sreetamdas thank you so much, thank you so much for fast response. I have been busy here with some random events. 

Yes, I tried your answer so I got an error massage saying my gh-pages does not build correctly and they asked me to remove any relative permalink. when I tried to visit my site at life.azzamsa.com, the browser said "no site here". but after removing relative permalink in config.yml now my site served correcty and fine at life.azzamsa.com

thank you so much, thank you.

I will repay your kindness by helping others
